### PR TITLE
Routers

### PR DESCRIPTION
**Pull request by danielamoreno699**
**In this project, I :**

- As per requirement I created a website consisting of 3 pages: Home, Calculator, and Quote, following the wireframes. For that purpose I used the Router package of react
- In the Router directory, I imported all the files and established for each one of them their own router.
- I linked the routers to the navbar , so the user can click on each item. 
- I also added basic styling

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
   return (
 
     <div className="App">
+      <MagicianRoutes />
+      {/* <Navbar />
+      <Home />
       <Calculator />
-      <Quotes />
+      <Quotes /> */}
     </div>
```
Comment (cynthiainga) [inline src/App.js]: - [ ] Please kindly delete the commented line of code to have a clean codebase 🙃 🙃

File: src/index.js [hunk 2]
```diff
@@ -1,3 +1,6 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (cynthiainga) [inline src/index.js]: - [ ] Please kindly do not disable linters, it's better to fix linters it helps to write a clean and good codebase 😉

File: src/routes/MagicianRoutes.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (cynthiainga) [inline src/routes/MagicianRoutes.js]: - [ ] Please kindly do not disable linters, it's better to fix linters it helps to write a clean and good codebase 😉

File: src/components/navbar.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (cynthiainga) [inline src/components/navbar.js]: - [ ] Please kindly do not disable linters, it's better to fix linters it helps to write a clean and good codebase 😉
- [ ] Also I noticed you disabled linters in your `children.js` file, kindly fix that as well 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @danielamoreno699, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Good design ✔️ 
- [ ] Routes are used correctly ✔️ 
- [ ] Correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — APPROVED** [review]
Hi @danielamoreno699 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Correct usage of gitflow
✔️ linter are allowed to run as expected


## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**